src/cascadia/WindowsTerminal/IslandWindow.cpp

```diff
@@ -311,6 +320,10 @@ void IslandWindow::Initialize()
             _taskbar = std::move(taskbar);
         }
     }
+
+    // Enable vintage opacity by removing the XAML emergency backstop, GH#603.
+    auto hr = TerminalTrySetTransparentBackground(true);
+    LOG_IF_FAILED(hr);

```

Comment:
probably don't need to log this one honestly; we don't need a bunch of error reports